README.md

```diff
@@ -40,16 +40,93 @@ The Rucio file catalog is much flatter than that of a usual `POSIX` filesystems
 - Fix Mac OS X mouting issues
 - Surely a lot more...
 
-## How to build
```

Comment:
General: around code blocks please leave a blank line. This is a md good practice!

Reply:
Oh Sure! I'll make sure I correct this here any in any documentation in the future